draft-ietf-anima-brski-prm.md

```diff
@@ -928,99 +984,102 @@ The fields contained in the `pledgevoucherrequesttrigger` are:
+##### JSON Agent-Signed Data {#prm-asd}
+
+The JSON Agent-Signed Data SHALL be a JSON document {{!RFC8259}} that MUST conform with the CDDL {{!RFC8610}} data model defined in {{prmasd_CDDL_def}}:
 
 ~~~~ cddl
   prmasd = {
-    "created": tdate,
+    "created-on": tdate,
     "serial-number": text
   }
 ~~~~
-{: #prmasd_CDDL_def title='CDDL for BRSKI-PRM Agent Signed Data' artwork-align="left" sourcecode-markers=“true”}
-
-The fields contained in the `prmasd` are:
+{: #prmasd_CDDL_def title='CDDL for BRSKI-PRM Agent Signed Data' artwork-align="left"}
 
-* `created-on`: creation date and time as standard date/time string as defined in {{!RFC3339}} 
+The `created-on` member SHALL contain the current date and time at tPVR creation as standard date/time string as defined in {{Section 5.6 of !RFC3339}}.
 
-* `serial-number`: product-serial-number in the X520SerialNumber field of the IDevID certificate of the pledge as string as defined in {{Section 2.3.1 of !RFC8995}}
+The `serial-number` member SHALL contain the product-serial-number of the pledge with which the Registrar-Agent assumes to communicate as string.
+The format MUST correspond to the X520SerialNumber field of IDevID certificates.
 
-{{prmasd_payload}} below shows an example for unsigned BRSKI-PRM Agent Signed Data in JSON syntax. 
+{{prmasd_payload}} below shows an example for the JSON Agent-Signed Data:
 
 ~~~~
 {
   "created-on": "2021-04-16T00:00:01.000Z",
   "serial-number": "callee4711"
 }
 ~~~~
-{: #prmasd_payload title="Data example for prmasd" artwork-align="left"}
+{: #prmasd_payload title="JSON Agent-Signed Data Example" artwork-align="left"}
 
-The JWS Protected Header of the `agent-signed-data` JWS structure MUST contain the following parameters (see {{asd_header}} for an example):
+##### JWS Protected Header
 
-* `alg`: algorithm type used to create the signature, e.g., `ES256` as defined in {{Section 4.1.1 of RFC7515}}
+The JWS Protected Header of the `agent-signed-data` member MUST contain the following standard Header Parameters as defined in {{RFC7515}}:
 
-* `kid`: base64-encoded bytes of the SubjectKeyIdentifier (the "KeyIdentifier" OCTET STRING value) of the EE (RegAgt) certificate.
+* `alg`: SHALL contain the algorithm type used to create the signature, e.g., `ES256`, as defined in {{Section 4.1.1 of RFC7515}}
+* `kid`: SHALL contain the base64-encoded bytes of the SubjectKeyIdentifier (the `KeyIdentifier` OCTET STRING value) of the Registrar-Agent LDevID certificate as defined in {{Section 4.2.1.2 of !RFC5280}}
+
+{{asd_header}} below shows an example for this JWS Protected Header:
 
 ~~~~
 {
   "alg": "ES256",
   "kid": "base64encodedvalue=="
 }
 ~~~~
-{: #asd_header title="Protected Header example inside agent-signed-data" artwork-align="left"}
-
-Note that at the time of receiving the PVR trigger, the pledge cannot verify the registrar LDevID certificate and has no proof-of-possession of the corresponding private key for the certificate.
```

Comment:
I was looking for it in section 7.1, but could not find the related statement

Reply:
Section 7.1.1 Request Artifact: Pledge Voucher-Request Trigger (tPVR) has

> The artifact is unsigned because at the time of receiving the tPVR, the pledge can verify neither certificate nor signature and can only accept the parameters provisionally until it receives the voucher as described in Section 7.6 (see Section 5.4).

Section 5.4 Agent-Proximity Assertion has

> In a similar fashion, the pledge MUST accept the Registrar-Agent LDevID certificate provisionally. See also Section 5 of [RFC8995] on "provisional state".

Reply:
For a clearer processing description I added the following after the text on "Upon receiving a valid tPVR":

> Note that the pledge has to provisionally accept the tPVR fields until it receives the voucher (see {{agt_prx}}).

Reply:
Agree to all above.